.github/workflows/run-ci-for-external-forks.yml

```diff
@@ -32,13 +35,16 @@ jobs:
           team: wrangler
 
       - name: Stop workflow if user is not a team member
-        if: ${{ steps.teamAffiliation.outputs.isTeamMember == false }}
+        if: ${{ steps.teamAffiliation.outputs.isTeamMember == 'false' }}
```

Comment:
Looks like this never worked before :grimacing:

[Reference from README on tspascoal/get-user-teams-membership](https://github.com/tspascoal/get-user-teams-membership/blob/ba78054988f58bea69b7c6136d563236f8ed2fc0/README.md?plain=1#L67)